tests/manage/test_ocs_336_346.py

```diff
@@ -1,57 +1,17 @@
 import logging
```

Comment:
By removing the setup and teardown you changed the test itself to something is not written in Polarion.
In addition, we need to separate between setup, test and teardown - this change cause all steps to be part of the test which is not good

Reply:
Updated with new PR https://github.com/red-hat-storage/ocs-ci/pull/239